src/main/java/jenkins/plugins/office365connector/HttpWorker.java

```diff
@@ -75,7 +76,7 @@ public void run() {
                 // uncomment to log what message has been sent
                 // log("Posted JSON: %s", data);
                 requestEntity = new StringRequestEntity(data, "application/json", StandardCharsets.UTF_8.name());
-            } catch (UnsupportedEncodingException e) {
+            } catch (Exception e) {
```

Comment:
I prefer to have even a few specified exceptions if needed than one generic

Reply:
Thanks, I've reverted this.